docs/testcases/TC06-004-pass.html

```diff
@@ -136,7 +136,7 @@ <h4 id="TCcode">Code</h4>
   								<h4 id="TCurl">Single-Page URL</h4>
   									<p><a href="../testfiles/TF06/06-004.html">https://section508coordinators.github.io/BaselineTestPages/testfiles/TF06/06-004.html</a></p>
   								<h4 id="TCframe">Embedded Code</h4>
-  									<iframe src="../testfiles/TF06/06-004.html" class="card shadow my-3"></iframe>
+  									<iframe title="embedded test file" src="../testfiles/TF06/06-004.html" class="card shadow my-3"></iframe>
```

Comment:
TC06-004-pass.html:

  - Minor issue: The code `<p>` value does not match the actual `<p>` value in the embedded test file. The difference is minor but different.